### ISSUE TITLE
Missing whitespaces in old CSS Grid after minification

### ISSUE THREAD

**Issue by motss** [jakubpawlowicz/clean-css#946]
### Precheck
 
* Do a quick search and make sure a bug has not yet been reported;
* do a quick check if the bug still exists in the latest patch version;
* finally, be nice and have fun!

### Environment

* clean-css version - `npm ls clean-css`: 4.1.2
* node.js version - `node -v`: 6.10.2
* operating system: macOS Sierra 10.12.4

### Configuration options

```js
var CleanCSS = require('clean-css');
new CleanCSS()
```

### Input CSS

```css
.grid-layout {
    display: -ms-grid;
    -ms-grid-columns: ( 1fr )[5];
    -ms-grid-rows: 1fr;
    -ms-grid-column-align: center;
    -ms-grid-row-align: center;
}
```

### Actual output CSS

A redundant whitespace being inserted in between `(1fr)` and `[5]`.

```css
.grid-layout{display:-ms-grid;-ms-grid-columns:(1fr) [5];-ms-grid-rows:1fr;-ms-grid-column-align:center;-ms-grid-row-align:center}
```

### Expected output CSS

`-ms-grid-columns` cannot have whitespace in between `(1fr)` and `[5]`.

```css
.grid-layout{display:-ms-grid;-ms-grid-columns:(1fr)[5];-ms-grid-rows:1fr;-ms-grid-column-align:center;-ms-grid-row-align:center}
```

### Simple demo to reproduce issue

__**Note: Old CSS Grid specs only available on IE11 and MS Edge.__

[jsbin demo](http://jsbin.com/ranituguda/edit?html,output)

**Comment by jakubpawlowicz** [comment]
I can see it too - fix is coming!

**Comment by motss** [comment]
@jakubpawlowicz Thank you for the quick response on this issue that is critical to me and my applications that leverage both old and new CSS Grid. Big thumbs up!!! 👍 🎉 😸

**Comment by jakubpawlowicz** [comment]
I'll push an update later today.

**Comment by jakubpawlowicz** [comment]
It's fixed in 4.1.3!
